lerna.json

```diff
@@ -1,5 +1,19 @@
 {
+  "$schema": "https://json.schemastore.org/lerna",
+  "changelogPreset": "conventional-changelog-conventionalcommits",
```

Comment:
nit: can further customise this later to how we like
- https://github.com/LavaMoat/LavaMoat/issues/636

Reply:
@leotm I'm not sure I understand what you're suggestion is

Reply:
i remember seeing some prettified changelog in the past so wondered how it's done

till spotted `changelogPreset` also takes an obj!

```ts
// https://github.com/lerna/lerna/blob/main/libs/core/src/lib/conventional-commits/constants.ts
export type ChangelogPresetConfig = string | { name: string; [key: string]: unknown };
export type BaseChangelogOptions = {
  changelogPreset?: ChangelogPresetConfig;
  changelogEntryAdditionalMarkdown?: string;
  // etc
```

pos since 6.2.0 (2022-12-13) looks like

- https://github.com/lerna/lerna/issues/3441
> The current schema only allow for a string based "changelogPreset", but lerna also supports an object syntax for this property.

but yea not important